ReactSkia/components/RSkComponentText.cpp

```diff
@@ -42,6 +42,24 @@ inline SkScalar yPosOffset(AttributedString attributedString, SkScalar paraHeigh
     return 0;
 }
 
+inline int getNumberOfLines(std::vector<LineMetrics>& metrics, int maxNumberOfLines, Float frameHeight) {
+    int i=0;
+    int minNumberOfLines = 0;
+    for(; i< metrics.size(); i++) {
+        if ((metrics[i]).fBaseline > frameHeight) {
+            minNumberOfLines = (metrics[i]).fLineNumber;
+            break;
+        }
+    }
+    if ((minNumberOfLines >= 0) && (maxNumberOfLines > 0)) {
+        if ((minNumberOfLines > 0) && (minNumberOfLines < maxNumberOfLines))
+            return minNumberOfLines;
+        else
+            return maxNumberOfLines;
+    }
+    return 0;
```

Comment:
this condition should probably be 

if ((minNumberOfLines > 0) && (minNumberOfLines < maxNumberOfLines))
            return minNumberOfLines;

return  maxNumberOfLines;

Reply:
Udated